.github/workflows/docat.yml

```diff
@@ -39,9 +39,33 @@ jobs:
         run: |
           python -m poetry run pytest
 
+  javascript-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          #cache: 'yarn'
```

Comment:
Why is this commented out?

I would suggest to either remove it or use it?

Reply:
on purpose as a reminder that we could look into caching - but i can also remove it